app/javascript/components/TeacherSetDetails.jsx

```diff
@@ -340,15 +340,15 @@ export default function TeacherSetDetails(props) {
             id="ts-order-set"
             textAlign="center"
             noSpace
-            level="two"
-            size="secondary"
+            level="h2"
+            size="heading5"
             text="Order Set"
           />
           <Heading
             id="ts-available-copies"
             textAlign="center"
-            size="callout"
-            level="four"
+            size="heading6"
+            level="h4"
             text={AvailableCopies()}
           />
```

Comment:
This text can instead be rendered using the `subtitle` prop in the `Heading` component above.

Reply:
@deepikagonuguntla This still needs to be changed?